demisto_sdk/commands/common/update_id_set.py

```diff
@@ -1376,6 +1428,7 @@ def re_create_id_set(id_set_path: Optional[str] = DEFAULT_ID_SET_PATH, pack_to_c
     new_ids_dict['Reports'] = sort(reports_list)
     new_ids_dict['Widgets'] = sort(widgets_list)
     new_ids_dict['Mappers'] = sort(mappers_list)
+    new_ids_dict['Packs'] = {key: packs_dict[key] for key in sorted(packs_dict.keys())}
```

Comment:
Why is it beneficial to sort it if it's going into an orderless dict anyway?

Reply:
you right, done